include/LightGBM/utils/common.h

```diff
@@ -248,7 +248,8 @@ inline static const char* Atof(const char* p, double* out) {
     if (cnt > 0) {
       std::string tmp_str(p, cnt);
       std::transform(tmp_str.begin(), tmp_str.end(), tmp_str.begin(), Common::tolower);
-      if (tmp_str == std::string("na") || tmp_str == std::string("nan")) {
+      if (tmp_str == std::string("na") || tmp_str == std::string("nan") ||
+          tmp_str == std::string("null") || tmp_str == std::string("NULL")) {
```

Comment:
line 250 already convert to lower case. So not need the  "NULL" .

Reply:
ok. i change it